keyboards/sawnsprojects/re65/info.json

```
@@ -0,0 +1,142 @@
+{
+    "keyboard_name": "RE65",
+    "maintainer": "Salane",
+    "manufacturer": "Mai The San",
+    "url": "",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0x534C",
+        "pid": "0x0C65",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true,
+        "rgblight": true,
+        "encoder": true,
+        "encoder_map": true
```

could encoder_map be enabled at keymap level please?
```suggestion
        "encoder": true
```

---

Correct 😁